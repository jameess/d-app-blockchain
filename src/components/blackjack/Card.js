import React, { Component } from 'react';


class Card extends Component {
    render() {
        const imageSource = `./img/${this.props.face}.png`;
        return (
            <div >
                {
                    this.props.hidden ?
                        (<img src="./img/hidden.png" alt="" />) :
                        (<img src={imageSource} alt="" />)
                }
            </div>
        );
    }
}

export default Card;
