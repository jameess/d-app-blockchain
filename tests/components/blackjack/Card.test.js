import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Card from '../../../src/components/blackjack/Card';

test('should render card component', () => {
    const wrapper = shallow(<Card />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});