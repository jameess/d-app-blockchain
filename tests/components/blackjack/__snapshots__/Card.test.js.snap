exports[`test should render NoMatch component 1`] = `
<div>
  <img
    alt=""
    src="./img/undefined.png" />
</div>
`;

exports[`test should render card component 1`] = `
<div>
  <img
    alt=""
    src="./img/undefined.png" />
</div>
`;
